src/views/AnalyticalFramework/FrameworkDetails/FrameworkDetailsForm/index.tsx

```diff
@@ -56,6 +54,14 @@ function FrameworkDetailsForm(props: Props) {
         frameworkGetPending,
     } = props;
 
+    const [
+        organizationOptions,
+        setOrganizationOptions,
+    ] = useState<BasicOrganization[] | undefined | null>(
+        analyticalFrameworkFromProps?.organizationDetails ?
+            [analyticalFrameworkFromProps.organizationDetails] : undefined,
```

Comment:
As framework details page will be mounted before analytical Framework details is fetched, this might not work all the time.
Lets move this logic inside a useState.